pkg/apis/acme/v1alpha2/types_issuer.go

```diff
@@ -43,18 +44,20 @@ type ACMEIssuer struct {
 
 	// PrivateKey is the name of a secret containing the private key for this
 	// user account.
+	// +kubebuilder:validation:Required
 	PrivateKey cmmeta.SecretKeySelector `json:"privateKeySecretRef"`
 
 	// Solvers is a list of challenge solvers that will be used to solve
 	// ACME challenges for the matching domains.
-	// +optional
+	// +kubebuilder:validation:MinItems=1
 	Solvers []ACMEChallengeSolver `json:"solvers,omitempty"`
 }
 
 // ACMEExternalAccountBinding is a reference to a CA external account of the ACME
 // server.
 type ACMEExternalAccountBinding struct {
 	// keyID is the ID of the CA key that the External Account is bound to.
+	// +kubebuilder:validation:Required
```

Comment:
This change doesn't seem to do anything - fields are already considered `required` if they are not `optional`